homework/src/components/Blog/Blog.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from "react";
+import { IPosts } from "src/types/types";
+import "./style.css";
+import Post from "../Post/Post";
+import TabsContainer from "../Tabs/TabsContainer";
+
+const Blog = () => {
+  const [posts, setPosts] = useState([]);
+  useEffect(() => {
+    const getPosts = async () => {
```

Comment:
It's better to have this function in a separate file called `helpers.ts`. Import it and use inside your component.